CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @mozilla/cloud-engineering-wg
```

Comment:
Oh, interesting--there isn't a cloud-engineering-wg in mozilla. I thought those work groups got copied everywhere.

Reply:
There is a cloud-engineering-wg in mozilla, so I don't understand what the problem was. Regardless, I switched it to mzcld-demo-wg which is one of the project collaborators.